go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	github.com/aliyun/aliyun-tablestore-go-sdk v1.7.7
 	github.com/apache/dubbo-go-hessian2 v1.11.1
 	github.com/apache/pulsar-client-go v0.9.0
-	github.com/apache/rocketmq-client-go/v2 v2.1.1-rc2
+	github.com/apache/rocketmq-client-go/v2 v2.1.0
```

Comment:
Instead can you please increase the version to `v2.1.1` and address any required changes?

I don't think going backwards is a good idea.

Reply:
v2.1.1 is unavailable due to compatibility problems

Reply:
I am submitting code to the RocketMQ community to address compatibility issues.

Reply:
Yes does not seem they maintained semantic versioning at all. Breaking change between 2.10 and 2.1.1, and even between 2.1.1-rc2 and 2.1.1. Very strange!